content/en/blog/_posts/2019-07-18-some-apis-are-being-deprecated.md

```diff
@@ -12,21 +12,45 @@ When APIs evolve, the old API is deprecated and eventually removed.
 
 The **v1.16** release will stop serving the following deprecated API versions in favor of newer and more stable API versions:
 
-* NetworkPolicy (in the **extensions/v1beta1** API group)
-  * Migrate to use the **networking.k8s.io/v1** API, available since v1.8.
-    Existing persisted data can be retrieved/updated via the **networking.k8s.io/v1** API.
-* PodSecurityPolicy (in the **extensions/v1beta1** API group)
+* NetworkPolicy in the **extensions/v1beta1** API version is no longer served
```

Comment:
I think a noun here would be better, what about
```suggestion
* NetworkPolicy in the **extensions/v1beta1** API version (no longer served)
```

Reply:
I find the parentheticals harder to read, but I don't feel that strongly